messaging/infinispan-grpc-kafka/src/test/java/io/quarkus/qe/books/BookServiceTest.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.qe.books;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.qe.containers.InfinispanTestResource;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.ContentType;
+
+// TODO: change to @QuarkusScenario once https://github.com/quarkus-qe/quarkus-test-framework/issues/292 is solved
```

Comment:
I don't think this issue should block using QuarkusScenario here. You can use `@Container` plus `::resource` for external resources to be included in the container instance.